src/routes/charts.js

```diff
@@ -462,6 +463,16 @@ function register(server, options) {
 
     if (!hasRegisteredDataPlugins) {
         events.on(event.GET_CHART_ASSET, async function({ chart, filename }) {
+            const fn = path.join(
```

Comment:
```suggestion
            const filePath = path.join(
```
`fn` is often the short form for `function` and I think a more expressive name is nice.